operators/generate-kas-fleetshard-olm-bundle.sh

```diff
@@ -108,6 +122,10 @@ else
     TEE=tee
 fi
 
+if [ "${UPDATE_KAS_INSTALLER_ENV}" = "true" ]; then
+    source ${KI_CONFIG}
+fi
```

Comment:
Does this need to be conditional? The `IMAGE_REPOSITORY_USERNAME` and `IMAGE_REPOSITORY_PASSWORD` vars used below are typically set in the env file.